From: "Rémi Bernon" <rbernon@codeweavers.com>
Subject: [PATCH 2/2] wine.inf: Start PlugPlay service on prefix initialization.
Message-Id: <20220104101303.2616686-2-rbernon@codeweavers.com>
Date: Tue,  4 Jan 2022 11:13:03 +0100
In-Reply-To: <20220104101303.2616686-1-rbernon@codeweavers.com>
References: <20220104101303.2616686-1-rbernon@codeweavers.com>

Device drivers may be installed and started on prefix initialization but
if PlugPlay is missing they won't be able to send plugplay notifications
until the prefix is restarted.

Signed-off-by: Rémi Bernon <rbernon@codeweavers.com>
---

On prefix initialization plugplay.exe service is not automatically
started, as the service has just been registered, and devices
notifications do not work at all.

 loader/wine.inf.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index c0251934dfc..3af8dabca64 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -178,7 +178,7 @@ AddService=RpcSs,0,RpcSsService
 AddService=Spooler,0,SpoolerService
 AddService=StiSvc,0,StiService
 AddService=TermService,0,TerminalServices
-AddService=PlugPlay,0,PlugPlayService
+AddService=PlugPlay,0x800,PlugPlayService
 AddService=FontCache3.0.0.0,0,WPFFontCacheService
 AddService=LanmanServer,0,LanmanServerService
 AddService=FontCache,0,FontCacheService
@@ -197,7 +197,7 @@ AddService=RpcSs,0,RpcSsService
 AddService=Spooler,0,SpoolerService
 AddService=StiSvc,0,StiService
 AddService=TermService,0,TerminalServices
-AddService=PlugPlay,0,PlugPlayService
+AddService=PlugPlay,0x800,PlugPlayService
 AddService=FontCache3.0.0.0,0,WPFFontCacheService
 AddService=LanmanServer,0,LanmanServerService
 AddService=FontCache,0,FontCacheService
@@ -216,7 +216,7 @@ AddService=RpcSs,0,RpcSsService
 AddService=Spooler,0,SpoolerService
 AddService=StiSvc,0,StiService
 AddService=TermService,0,TerminalServices
-AddService=PlugPlay,0,PlugPlayService
+AddService=PlugPlay,0x800,PlugPlayService
 AddService=FontCache3.0.0.0,0,WPFFontCacheService
 AddService=LanmanServer,0,LanmanServerService
 AddService=FontCache,0,FontCacheService
@@ -235,7 +235,7 @@ AddService=RpcSs,0,RpcSsService
 AddService=Spooler,0,SpoolerService
 AddService=StiSvc,0,StiService
 AddService=TermService,0,TerminalServices
-AddService=PlugPlay,0,PlugPlayService
+AddService=PlugPlay,0x800,PlugPlayService
 AddService=FontCache3.0.0.0,0,WPFFontCacheService
 AddService=LanmanServer,0,LanmanServerService
 AddService=FontCache,0,FontCacheService

-- 
2.34.1

